metal/resource_metal_device.go

```diff
@@ -422,7 +454,7 @@ func resourceMetalDeviceCreate(d *schema.ResourceData, meta interface{}) error {
 func resourceMetalDeviceRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*packngo.Client)
 
-	device, _, err := client.Devices.Get(d.Id(), &packngo.GetOptions{Includes: []string{"project"}})
+	device, _, err := client.Devices.Get(d.Id(), &packngo.GetOptions{Includes: []string{"project", "metro"}})
```

Comment:
I wonder if we should be defensive and include `facility` too, just incase the default inclusion of facility changes in the future.

Reply:
good idea. done in 8e5bc05